app/services/abandon_overdue_solution_mentorships.rb

```diff
@@ -0,0 +1,10 @@
+class AbandonOverdueSolutionMentorships
```

Comment:
Do we run this in a rake task via cron? I think we're missing the rake task.

Reply:
We will do but I'm going to run it manually for a bit first before we cron it.